lib/cea/cea608_data_channel.js

```diff
@@ -420,22 +447,20 @@ shaka.cea.Cea608DataChannel = class {
    * @private
    */
   controlTr_() {
-    const buf = this.text_;
-
-    // Window for text mode is the entire screen i.e. all of the rows.
-    buf.setScrollSize(shaka.cea.Cea608Memory.CC_ROWS);
-    buf.eraseBuffer();
+    this.text_.reset();
     this.controlRtd_();  // Put into text mode.
   }
 
   /**
    * Handles RTD - Resume Text Display.
-   * Resumes text mode. Mo need to force anything out, because Text Mode doesn't
-   * affect the current display. Also, this decoder does not emit TEXTn anyway.
+   * Resumes text mode. No need to force anything out, because Text Mode doesn't
+   * affect current display. Also, this decoder does not emit Text Mode anyway.
    * @private
    */
   controlRtd_() {
-    this.curbuf = this.text_;
+    shaka.log.alwaysWarn('CEA-608 text mode entered, but is unsupported');
```

Comment:
I would also suggest setting a flag on `this` so you only issue the warning once, instead of once per packet or something.